Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM node:13-alpine AS build
+
+# Install git for cloning dependencies
+RUN apk add git python make g++
+
+# Set workdir
+WORKDIR /app
+
+# Add package.json and yarn.lock
+ADD package.json yarn.lock ./
+
+# Install dependencies
+RUN yarn install
+
+# Copy resources
+ADD . .
+
+# Build static site
+RUN yarn generate
+
+# Deploy to nginx
+FROM nginx:latest
```

Comment:
nit - `nginx:alpine` is roughly 5x smaller

Reply:
Oops yes, gonna fix. Shouldn't have copy-pasted there, oof.